modules/exploits/windows/local/bits_ntlm_token_impersonation.rb

```diff
@@ -0,0 +1,162 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/post/windows/reflective_dll_injection'
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = NormalRanking
+
+  include Msf::Post::File
+  include Msf::Post::Windows::Priv
+  include Msf::Post::Windows::Process
+  include Msf::Post::Windows::ReflectiveDLLInjection
+
+  def initialize(info = {})
+    super(update_info(info, {
+        'Name'           => 'SYSTEM token impersonation through NTLM bits authentication on missing WinRM Service.',
+        'Description'    => %q(
+        This module exploit BITS behavior which tries to connect to the
+        local Windows Remote Management server (WinRM) every times it
+        starts. The module launches a fake WinRM server which listen on
+        port 5985 and triggers BITS. When BITS starts, it tries to
+        authenticate to the Rogue WinRM server, which allows to steal a
+        SYSTEM token. This token is then used to launch a new process
+        as SYSTEM user. In the case of this exploit, powershell.exe
+        is launched as SYSTEM and it downloads/executes an evil powershell
+        script on a second local http server. Thus, this exploit don't
+        write any file on the disk.
+
+        WARNING:
+
+        - As this exploit runs 2 services on the target
+        (Fake WinRM on port 5985 and HTTP on port 80), a firewall popup
+        may appear on target screen. Thus, this exploit is not completely
+        silent.
+
+        - A powershell windows flashes on target screen
```

Comment:
We have metadata for this that should be added to the module (`SCREEN_EFFECTS`), see: https://github.com/rapid7/metasploit-framework/wiki/Definition-of-Module-Reliability,-Side-Effects,-and-Stability.

Reply:
Fixed by commit [e4272e3c88c6e8b8dbe7153f2fa02336a66451a9](https://github.com/C4ssandre/metasploit-framework/commit/e4272e3c88c6e8b8dbe7153f2fa02336a66451a9)